src/pages/api/contacts-poe/index.controller.test.ts

```diff
@@ -0,0 +1,128 @@
+import { testApiHandler } from 'next-test-api-route-handler';
+import nock from 'nock';
+
+import { enregistrerContactPOEHandler } from '~/pages/api/contacts-poe/index.controller';
+import { ErrorHttpResponse } from '~/server/errors/errorHttpResponse';
+
+describe('enregistrerContactPOEHandler', () => {
+	describe('quand la méthode n‘est pas un POST', () => {
+		it('retourne une erreur 406', async () => {
+			await testApiHandler<void | ErrorHttpResponse>({
+				handler: (req, res) => enregistrerContactPOEHandler(req, res),
+				test: async ({ fetch }) => {
+					const res = await fetch({ method: 'GET' });
```

Comment:
J'ai l'impression que tu testes juste le cas GET et pas tous ce qui n'est pas un POST

Reply:
J'ajoute un it.each sur les 2 méthodes manquantes :)